mmv1/products/tpuv2/QueuedResource.yaml

```diff
@@ -31,17 +31,11 @@ async:
   operation:
     base_url: '{{op_id}}'
   result:
-    path: 'response'
     resource_inside_response: true
   status:
-    path: 'done'
     complete: true
-    allowed:
       - true
       - false
```

Comment:
```suggestion
```

Reply:
ah yeah, just ran into this after running the serializer locally